pulsar-client-cpp/tests/CMakeLists.txt

```diff
@@ -17,12 +17,12 @@
 # under the License.
 #
 
-find_library(GTEST_LIBRARY_PATH gtest)
+find_library(GMOCK_LIBRARY_PATH gmock)
```

Comment:
We might also need to update the Docker images used for building the C++ libray as well as the documentation.